### PR TITLE
Use new blueprint testing API

### PR DESCRIPTION
**Pull request by Turbo87**
This PR modifies the blueprint tests to use the new blueprint testing API introduced in https://github.com/ember-cli/ember-cli-blueprint-test-helpers/pull/45.

### CODE REVIEW

File: node-tests/blueprints/adapter-test.js [hunk 1]
```diff
@@ -1,133 +1,107 @@
...
+      .then(() => modifyPackages([
+        {name: 'ember-cli-qunit', delete: true},
+        {name: 'ember-cli-mocha', dev: true}
+      ]))
+      .then(() => emberGenerateDestroy(args, _file => {
+        expect(_file('tests/unit/adapters/foo-test.js'))
+          .to.contain('import { describeModule, it } from \'ember-mocha\';')
+          .to.contain('describeModule(\n  \'adapter:foo\',')
+          .to.contain('expect(adapter)' +
+          '.to.be.ok;');
```
Comment (pangratz) [inline node-tests/blueprints/adapter-test.js]: Can you combine this and the line above into 1 line, so it's consistent with the test for the transform test?

File: node-tests/blueprints/model-test.js [hunk 2]
```diff
@@ -43,144 +40,110 @@ describe('Acceptance: generate and destroy model blueprints', function() {
...
+      .then(() => modifyPackages([
+        {name: 'ember-cli-qunit', delete: true},
+        {name: 'ember-cli-mocha', dev: true}
+      ]))
+      .then(() => emberGenerateDestroy(args, _file => {
+        expect(_file('tests/unit/models/foo-test.js'))
+          .to.contain('import { describeModel, it } from \'ember-mocha\';')
+          .to.contain('describeModel(\n  \'foo\',')
+          .to.contain('expect(model)' +
+          '.to.be.ok;');
```
Comment (pangratz) [inline node-tests/blueprints/model-test.js]: same same

File: node-tests/blueprints/serializer-test.js [hunk 3]
```diff
@@ -1,135 +1,109 @@
...
+        {name: 'ember-cli-mocha', dev: true}
+      ]))
+      .then(() => emberGenerateDestroy(args, _file => {
+        expect(_file('tests/unit/serializers/foo-test.js'))
+          .to.contain('import { describeModel, it } from \'ember-mocha\';')
+          .to.contain('describeModel(\n  \'foo\',')
+          .to.contain('Unit | Serializer | foo')
+          .to.contain('needs: [\'serializer:foo\']')
+          .to.contain('expect(serializedRecord)' +
+          '.to.be.ok;');
```
Comment (pangratz) [inline node-tests/blueprints/serializer-test.js]: same same

**Comment by pangratz** [comment]
:heart_eyes: New syntax looks great!

Another stylistic nitpick, but totally doable in another PR: I think we should replace the string to use `""` so we can get rid of the `'bla(\'asd\');'` escapes to a more readable `"bla('as');"`... what do you think?

File: node-tests/blueprints/adapter-test.js (hunk 1 above)
Comment (Turbo87) [inline node-tests/blueprints/adapter-test.js]: whoopsie, that broke during some multi-caret editing 😉

**Comment by Turbo87** [comment]
@pangratz fixed the style issues

**Comment by pangratz** [comment]
Thanks @Turbo87 :tada:

**Pull request merged**